index.html

```diff
@@ -9,85 +9,85 @@
 				</div>
-				<div class="reference">Dachstein Mountains, Obertraun, Austria by Kaidi Guo</div>
+				<div class="slide__reference">Dachstein Mountains, Obertraun, Austria by Kaidi Guo</div>
 			</div>
 			<div class="slide">
-				<div class="title" onclick="showReference()">
+				<div class="slide__title" onclick="showReference()">
 					<h3></h3>
 				</div>				
 				<img src="https://images.unsplash.com/photo-1516524609276-3cb95aa22190?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=3d79a0b6dc9683effb116d7e9763bfa2&auto=format&fit=crop&w=1225&q=80" alt="Tõrva, Estonia by Jaak Horn">
-				<div class="paragraph">
+				<div class="slide__paragraph">
 					<p></p>
 				</div>
-				<div class="reference">Tõrva, Estonia by Jaak Horn</div>
+				<div class="slide__reference">Tõrva, Estonia by Jaak Horn</div>
 			</div>
 			<div class="slide">
-				<div class="title" onclick="showReference()">
+				<div class="slide__title" onclick="showReference()">
 					<h3>Lutry, Switzerland</h3>
 				</div>
 				<img src="https://images.unsplash.com/photo-1517483141275-296bb15890b2?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=f6fb898eac03ab9abd1137fa691ebe82&auto=format&fit=crop&w=1051&q=80" alt="Lutry, Switzerland by Olivier Fahrni">
-				<div class="paragraph">
+				<div class="slide__paragraph">
 					<p>Lutry is a municipality in the Swiss canton of Vaud, located in the Lavaux-Oron.</p>
 				</div>
-				<div class="reference">Lutry, Switzerland by Olivier Fahrni</div>
+				<div class="slide__reference">Lutry, Switzerland by Olivier Fahrni</div>
 			</div>
 			<div class="slide">
-				<div class="title" onclick="showReference()">
+				<div class="slide__title" onclick="showReference()">
 					<h3>Mirror Lake Trail - Tosemite National Park</h3>
 				</div>
 				<img src="https://images.unsplash.com/photo-1480499484268-a85a2414da81?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=6d765c5039bd6d36022739e3a6552c44&auto=format&fit=crop&w=1055&q=80" alt="Mirror Lake, Yosemite Valley, United States by Quentin Dr">
-				<div class="paragraph">
+				<div class="slide__paragraph">
 					<p>From the shuttle stop (#17), a paved trail leads directly to Mirror Lake. Which is in the Adirondack Mountains in northern New York in the United States. The lake is approximately 124 acres, with a watershed area of 741 acres.</p>
 				</div>
-				<div class="reference">Mirror Lake, Yosemite Valley, United States by Quentin Dr</div>
+				<div class="slide__reference">Mirror Lake, Yosemite Valley, United States by Quentin Dr</div>
+			</div>
+			<div class="thumbnails"> 
+				<div class="thumbnails__imgs thumbnails__imgs--actual" onclick="showSlide(idx, idx = 1)"><img src="https://images.unsplash.com/photo-1523977843538-0650e8cb2e7d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=a43d0d3984b7a6e57716c15ee6fe5797&auto=format&fit=crop&w=1050&q=80"></div> 
```

Comment:
I have a little concern here.. why all of the thumbnails have the --actual modifier? should this modifier be only for the one that is showing actually?

Reply:
Sorry my bad I thought I have eliminated those